### DISCUSSION TITLE
Develop a data model for common diseases

### DISCUSSION THREAD

**Discussion by monicacecilia** [monarch-initiative/mondo#3861] [Bigger project]
- Work with key collaborators and stakeholders (e.g., ClinGen Clinical groups) to define a common model for disease annotation.
- Simplify the model to be boolean. 
- Define an MVP for release of v0.1

**Comment by nicolevasilevsky** [comment]
See slides for more background:
https://docs.google.com/presentation/d/1AeyxK5RNjmHOdVGU8E9e4IUe9yGm0wlaiG2f_tSP89I/edit#slide=id.p

need to do a show and tell:
- POET tool/HPO annotation model
- Koza axioms
- Mondo axioms
- disease annotation model

How are these used by downstream tools?

**Comment by nicolevasilevsky** [comment]
this is in progress and a much bigger project than can be captured in one ticket. I will move this to the discussion board.
